modules/admin-ui/src/main/webapp/scripts/shared/services/wizards/new-event/metadata.js

```diff
@@ -97,7 +97,16 @@ angular.module('adminNg.services')
 
           params.presentableValue = presentableValue;
         } else {
-          params.presentableValue = params.collection[value];
+          if (params.collection.hasOwnProperty(value)) {
+            params.presentableValue = params.collection[value];
+          } else {
+            // this should work in older browsers, albeit looking clumsy
+            var matchingKey = Object.keys(params.collection)
+              .filter(function(key) {return params.collection[key] === value;})[0];
+            params.presentableValue = params.type === 'ordered_text'
+              ? JSON.parse(matchingKey)['label']
+              : matchingKey;
+          }
```

Comment:
Previously, it was assumed that if the metadata object had a collection property and the value property did not have an array assigned to it, the value property would have a value that is named like a property of said collection. (I realise that this is worded confusingly, but that happens when a property is named *value*.) As you can see in the following screenshot, this would hold true for certain metadata fields having this collection property, but not for all of them. Those fields that did not make it to the summary had a **value** named like the value of the value property, not a property. To make the mess complete, collection elements with the type *ordered_text* are passed with a JSON string literal as property name, so to get to the desired *presentable value* such a property name would have to be parsed first, allowing to extract the wanted value from the parsed object.

<img width="899" alt="metadata" src="https://user-images.githubusercontent.com/16406401/46999551-461bf580-d126-11e8-9428-752e52657566.png">

Obviously, this is far from a perfect solution, but these inconsistencies are apparently rooted so deep that I dare not jump down into this rabbit hole.

Reply:
God, reading this could cause a stroke. But yeah, that's not really the topic of this PR.